src/HtmlRenderer.php

```diff
@@ -142,4 +142,12 @@ public function renderBlocks($blocks, $inTightList = false)
 
         return implode($this->options['blockSeparator'], $result);
     }
+
+    /**
+     * @return Environment
+     */
+    public function getEnvironment()
+    {
+        return $this->environment;
+    }
```

Comment:
I'm just worried about adding and supporting new public methods  solely for testing purposes (no strong use-case otherwise).